WMF Framework/Remote Notifications/RemoteNotificationsController.swift

```diff
@@ -61,6 +61,24 @@ import CocoaLumberjackSwift
             return []
         }
     }
+    
+    public func inboxCount() -> Int {
+        assert(Thread.isMainThread)
+        
+        guard let viewContext = self.viewContext else {
+            DDLogError("Failure fetching notifications from persistence: missing viewContext")
+            return 0
+        }
+        
+        let fetchRequest: NSFetchRequest<RemoteNotification> = RemoteNotification.fetchRequest()
+        
+        do {
+            return try viewContext.fetch(fetchRequest).count
```

Comment:
Try `return try viewContext.count(for: fetchRequest)` instead - I think it's more efficient under the hood than first fetching an array of notifications and getting the count.